### ISSUE TITLE
Add Visualization instructions to documentations

### ISSUE THREAD

**Issue by ljhwang** [geodynamics/best_practices#38]
Should we explicitly add that the documents should include recommendation, instructions and/or examples of how to visualize the results?

Would this also be part of workflows?

**Comment by gassmoeller** [comment]
I agree that would be useful, but maybe falls under documentation? E.g. Standard 5.i or 5.iv. could be extended to mention visualization.
